src/drivers/SpiMaster.cpp

```diff
@@ -131,8 +131,8 @@ void SpiMaster::OnEndEvent() {
   if (s > 0) {
     auto currentSize = std::min((size_t) 255, s);
     PrepareTx(currentBufferAddr, currentSize);
-    currentBufferAddr += currentSize;
-    currentBufferSize -= currentSize;
+    currentBufferAddr = currentBufferAddr + currentSize;
```

Comment:
What change away from the shorter `+=` operator don't in place increment to the longer version (that most probably will be optimized to the same code but I'm curious)

Reply:
I'm not sure why, but the compiler complains about `+=` on `volatile` variables:

```
/home/jf/git/PineTime/src/drivers/SpiMaster.cpp: In member function 'void Pinetime::Drivers::SpiMaster::OnEndEvent()':
/home/jf/git/PineTime/src/drivers/SpiMaster.cpp:134:23: error: compound assignment with 'volatile'-qualified left operand is deprecated [-Werror=volatile]
  134 |     currentBufferAddr += currentSize;
      |     ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
/home/jf/git/PineTime/src/drivers/SpiMaster.cpp:135:23: error: compound assignment with 'volatile'-qualified left operand is deprecated [-Werror=volatile]
  135 |     currentBufferSize -= currentSize;
      |     ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
```

It probably wants to make it explicit that the left operand is evaluated twice?